robottelo/constants/__init__.py

```diff
@@ -33,12 +33,13 @@
 
 
 DISTRO_DEFAULT = 'rhel7'
-DISTROS_SUPPORTED = ['rhel6', 'rhel7', 'rhel8', 'rhel9']
+DISTROS_SUPPORTED = ['rhel6', 'rhel7', 'rhel8', 'rhel9', 'rhel10']
```

Comment:
Is `rhel6` still supported?

Reply:
Nope! https://github.com/Katello/katello/blob/a739b7104bde511b2a77eead93c6207cc94943a2/app/models/katello/rhel_lifecycle_status.rb#L49